projects/state-mach-regex/extractlinks/extractlinks.js

```diff
@@ -10,11 +10,20 @@ if (args.length != 1) {
 const filename = args[0];
 
 // !!!! IMPLEMENT ME
-
 // Read file
+fs.readFile(filename, "utf8", function read(err, data) {
+    if (err) {
+        throw err;
+    }
+    const http = data.match(/((http:|https:).*?)"/g).map(h => h.replace(/\"/g, ""))
```

Comment:
Instead of `map`/`replace`-ing the `"` character, we can just use regular expression to exclude them.

```
https?:[^'"]+\.[^'"]+
```